bonito/basecaller.py

```diff
@@ -86,8 +92,8 @@ def argparser():
     parser.add_argument("--weights", default="0", type=str)
     parser.add_argument("--beamsize", default=5, type=int)
     parser.add_argument("--batchsize", default=1, type=int)
-    parser.add_argument("--chunksize", default=0, type=int)
-    parser.add_argument("--overlap", default=0, type=int)
+    parser.add_argument("--chunksize", default=0, type=check_chunksize)
+    parser.add_argument("--overlap", default=2988, type=int)
```

Comment:
As the chunk size and the overlap are tied to a models stride I think the best place to define them is in the models `config.toml`.

Reply:
Good point. Proposed to use
```
[inference]
overlap = 2988
```
section for default overlap